Lib/test/test_array.py

```diff
@@ -341,8 +341,6 @@ def test_iterator_pickle(self):
             a.fromlist(data2)
             self.assertEqual(list(it), [])
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
```

Comment:
It's interesting how this "overflow" behavior is actually correct with how iterators work in CPython